src/SprykerEco/Zed/UnzerApi/Persistence/Propel/Schema/spy_unzer_api.schema.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<database xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed" xsi:noNamespaceSchemaLocation="http://static.spryker.com/schema-01.xsd" namespace="Orm\Zed\UnzerApi\Persistence" package="src.Orm.Zed.UnzerApi.Persistence">
+
+    <table name="spy_payment_unzer_api_log" idMethod="native" allowPkInsert="true" isCrossRef="true">
+        <column name="id_payment_unzer_api_log" required="true" type="INTEGER" primaryKey="true" autoIncrement="true"/>
+        <column name="request_type" required="true" type="VARCHAR" />
+        <column name="url" required="true" type="VARCHAR" />
+        <column name="request" required="true" type="LONGVARCHAR"/>
```

Comment:
Propel maps `LONGVARCHAR` to native DB type `TEXT` which has different sizes for PGSQL and MySQL, please, check this [RFC](https://spryker.atlassian.net/wiki/spaces/RFC/pages/1451557821/ACCEPTED+RFC+Replacing+LONGVARCHAR+usages) for more info. 
Please, make sure, that `LONGVARCHAR` is enough for your case.

Reply:
`LONGVARCHAR` is enough